providers/lmgtfy/lmgtfy.go

```diff
@@ -0,0 +1,20 @@
+package google
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("lmgtfy", &LMGTFYProvider{})
+}
+
+// LMGTFYProvider implements the Provider interface.
+type LMGTFYProvider struct{}
```

Comment:
Check out the latest providers for the new naming convention. I think you will like it. =)